manager.gd

```diff
@@ -122,6 +120,7 @@ func benchmark(test_ids: Array[TestID], return_path: String) -> void:
 	if return_path:
 		get_tree().change_scene_to_file(return_path)
 	else:
+		get_tree().queue_delete(get_tree())
```

Comment:
This fixes a warning on exit: `ObjectDB Instances Leaked At Exit`.
I'm not sure if this is the proper way to fix it though.

Reply:
only fixes it in certain situations, so maybe just remove that line